src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx

```diff
@@ -35,7 +34,9 @@ const sortRisksByDate = (a: Risk, b: Risk) => {
 const RiskLog: React.FC<RiskLogProps> = ({ projectId, wbsNum }) => {
   const history = useHistory();
   const auth = useAuth();
-  const { userId } = auth.user!;
+  const { userId, role } = auth.user!;
+
+  const hasPermissions = role === 'ADMIN' || role === 'APP_ADMIN' || role === 'LEADERSHIP';
```

Comment:
you're missing the part where they're any of these roles OR they're the PM or PL of the project